scripts/run-spark.py

```diff
@@ -136,6 +152,7 @@ def generate_input(args):
 
 
 def sort_main(args):
+    run("hdfs dfs -rm -r -f /terasort/output")
     spark_args = get_spark_args(args)
```

Comment:
And add a comment here that we don't need to set Spark parallelism here because it derives from the number of partitions of the input, which is already determined from the `TeraGen` step.